docs/extend/plugins.md

```diff
@@ -34,7 +34,7 @@ Follow the instructions in the plugin's documentation.
 The sections below provide an inexhaustive overview of available plugins.
 
 <style>
-#content tr td:first-child { white-space: nowrap;}
+#DocumentationText  tr td:first-child { white-space: nowrap;}
```

Comment:
hm, let me try. Probably some style overriding it (there's some nasty CSS in the skin that's used 😞 )